auth0/resource_auth0_organization.go

```diff
@@ -74,83 +69,32 @@ func newOrganization() *schema.Resource {
 						"assign_membership_on_login": {
 							Type:     schema.TypeBool,
 							Optional: true,
+							Default:  false,
```

Comment:
This default is very important to correctly figure out the differences on the hashed sets.